module/files/helper.js

```diff
@@ -172,20 +189,53 @@ module.exports = class FilesHelper {
 
         // Return success response with the signed URLs
         return resolve({
-            success: true,
-            message: constants.apiResponses.URL_GENERATED,
-            result: signedUrls
+          success: true,
+          message: constants.apiResponses.URL_GENERATED,
+          result: signedUrls,
         });
-        
       } catch (error) {
         return reject({
           success: false,
           message: constants.apiResponses.FAILED_PRE_SIGNED_URL,
-          error: error
-        })
+          error: error,
+        });
       }
-    })
+    });
   }
+
+  /**
+   * Upload and get Download Url for a file.
+   * @method
+   * @name upload
```

Comment:
@ankitpws Please update the function signature.

Reply:
Done